tests/test_prompt.py

```diff
@@ -154,49 +160,54 @@ def test_copy_default_advertised(tmp_path_factory, spawn, name):
         ("[% if question_1 %]FALSE[% endif %]", False),
     ),
 )
-def test_when(tmp_path_factory, question_1, question_2_when, spawn, asks):
+def test_when(
+    tmp_path_factory: pytest.TempPathFactory,
+    spawn: Spawn,
+    question_1: Union[str, int, float, bool],
+    question_2_when: Union[bool, str],
+    asks: bool,
+) -> None:
     """Test that the 2nd question is skipped or not, properly."""
-    template, subproject = (
-        tmp_path_factory.mktemp("template"),
-        tmp_path_factory.mktemp("subproject"),
-    )
-    questions = {
-        "_envops": BRACKET_ENVOPS,
-        "_templates_suffix": SUFFIX_TMPL,
-        "question_1": question_1,
-        "question_2": {"default": "something", "type": "yaml", "when": question_2_when},
-    }
+    src, dst = map(tmp_path_factory.mktemp, ("src", "dst"))
     build_file_tree(
         {
-            template / "copier.yml": yaml.dump(questions),
-            template
-            / "[[ _copier_conf.answers_file ]].tmpl": "[[ _copier_answers|to_nice_yaml ]]",
+            (src / "copier.yml"): yaml.dump(
+                {
+                    "_envops": BRACKET_ENVOPS,
+                    "_templates_suffix": SUFFIX_TMPL,
+                    "question_1": question_1,
+                    "question_2": {
+                        "type": "yaml",
+                        "default": "something",
+                        "when": question_2_when,
+                    },
+                }
```

Comment:
Do you agree with this change? I think that inlining the content of `copier.yml` is more concise and consistent with the rest of the test suite.

Reply:
Putting risk/value into balance, I think it's better to undo it.

Reply:
Oh well ... Done. :shrug: